src/definitions.d.ts

```diff
@@ -54,6 +54,7 @@ export interface Theme {
     lightSchemeTextColor: string;
     scrollbarColor: '' | 'auto' | string;
     selectionColor: '' | 'auto' | string;
+    styleStandardElements: boolean;
```

Comment:
Maybe it's worth renaming to `styleFormControls` (understood by web devs) or `styleSystemControls` (maybe more clear for users)? Maybe you will have some other ideas.